modules/@apostrophecms/video-widget/ui/src/index.js

```diff
@@ -57,7 +57,8 @@ export default () => {
           if (result.width && result.height) {
             inner.style.width = '100%';
             resizeVideo(inner);
-            // If we need to iniitally size the video, also resize it on window resize.
+            // If we need to initially size the video, also resize it on window
+            // resize.
```

Comment:
I got so much grief for sticking to 80 column line lengths in our PHP days. I So I hate to give you grief for sticking to 80 column line lengths, and I'm going to happily accept this PR, but if we find it requires bigger changes to other files you touch, maybe we should rethink the requirement or do it all in one swell foop.

Reply:
One weekend I tried to work out a utility to take blocks of text and return them at 80 characters lines. Punctuation and indentation complicates things. So I approach this a bit at a time.